ts_src/payments/p2wsh.ts

```diff
@@ -59,6 +69,9 @@ export function p2wsh(a: Payment, opts?: PaymentOpts): Payment {
   const _rchunks = lazy.value(() => {
     return bscript.decompile(a.redeem!.input!);
   }) as StackFunction;
+  const _rochunks = lazy.value(() => {
```

Comment:
What does this `_rochunks` stand for? Read output chunks?

Maybe we could use a more descriptive name here?

Reply:
Also why does it need to be lazily evaluated?